client/my-sites/customer-home/cards/tasks/site-setup-list/get-task-data.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { domainManagementEdit, domainManagementList } from 'my-sites/domains/paths';
+import { requestSiteChecklistTaskUpdate } from 'state/checklist/actions';
+import { launchSiteOrRedirectToLaunchSignupFlow } from 'state/sites/launch/actions';
+import { localizeUrl } from 'lib/i18n-utils';
+import { openSupportArticleDialog } from 'state/inline-support-article/actions';
+
+export const getTaskData = ( task, { menusUrl, siteId, siteSlug, taskUrls, userEmail } = {} ) => {
+	let taskData = {};
+	switch ( task.id ) {
+		case 'domain_verified':
+			taskData = {
+				todo: {
+					timing: 2,
+					title:
+						task.unverifiedDomains.length === 1
+							? translate( 'Verify the email address for %(domainName)s', {
+									args: { domainName: task.unverifiedDomains[ 0 ] },
+							  } )
+							: translate( 'Verify the email address for your domains' ),
+					description: translate(
+						'We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working.'
+					),
+					actionUrl:
+						task.unverifiedDomains.length === 1
+							? domainManagementEdit( siteSlug, task.unverifiedDomains[ 0 ] )
+							: domainManagementList( siteSlug ),
+					actionText: translate( 'Verify' ),
+				},
+				completed: {
+					title: translate( 'You verified your email for your doamins.' ),
+					description: translate(
+						'You can always update your domain information on the Domains page.'
+					),
+					actionUrl: `/domains/manage/${ siteSlug }`,
+					actionText: translate( 'Manage domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Manage my domains' )` *ES Score: 7*